### DISCUSSION TITLE
Now that we have SDXL 1.0 ... How do we load the refiner???

### DISCUSSION THREAD

**Discussion by jdc4429** [Comfy-Org/ComfyUI#1000] [General]
Hey all,

How do we load the refiner when using SDXL 1.0 or .9???
Does it get placed in the same directory as the models (checkpoints)? or in Diffusers???

Also I tried using a more advanced workflow which requires a VAE but when I try using SDXL 1.0 with the supplied VAE I just get errors.. 

This is such a great front end.. just needs a few little things... Like the API to work the same way.. cannot queue from API but can in program...  

regards,

jdc4429

**Comment by jdc4429** [comment]
Figured out a way.. Found a really cool workflow by Sytan...

https://github.com/SytanSD/Sytan-SDXL-ComfyUI

**Reply by B34STW4RS** [reply]
Thanks for the hot tip pal, also just decided that I felt I wanted more control over my generations and decided to give Comfy a go, this really helped out.
